android/app/src/main/java/app/organicmaps/editor/OsmLoginBottomFragment.java

```diff
@@ -0,0 +1,115 @@
+package app.organicmaps.editor;
+
+import android.app.Dialog;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ProgressBar;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.android.material.bottomsheet.BottomSheetBehavior;
+import com.google.android.material.bottomsheet.BottomSheetDialog;
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
+import com.google.android.material.textfield.TextInputEditText;
+
+import app.organicmaps.R;
+import app.organicmaps.util.Constants;
+import app.organicmaps.util.InputUtils;
+import app.organicmaps.util.UiUtils;
+import app.organicmaps.util.Utils;
+import app.organicmaps.util.concurrency.ThreadPool;
+import app.organicmaps.util.concurrency.UiThread;
+
+public class OsmLoginBottomFragment extends BottomSheetDialogFragment {
```

Comment:
```suggestion
public class OsmLoginBottomFragment extends BottomSheetDialogFragment
{
```

Please update Android Studio formatter according to our code style and fix it everywhere.